gnucash/gnome/gnc-plugin-page-invoice.c

```diff
@@ -324,6 +359,12 @@ static action_toolbar_labels invoice_action_tooltips[] = {
     {NULL, NULL},
 };
 
+static action_toolbar_labels invoice_action_layout_tooltips[] = {
+    {"WindowsSaveLayoutAction", N_("Save this layout for the default Customer Document")},
+    {"WindowsResetLayoutAction", N_("Reset column widths and remove the default layout for Customer Documents")},
```

Comment:
"Reset default layout for all customer invoices and credit notes back to built-in default and update current page accordingly"

Reply:
OK, your text is much better.